### PR TITLE
Bike details tests android

### PR DESCRIPTION
**Pull request by StefanaIvascu**

### CODE REVIEW

File: keywords/common/bike_details_ui_step_def.robot [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
+
+
 *** Keywords ***
 
 the user adds a bike
     Click Element     ${Add-first-bike}
     Sleep   1s
-    Click Element     ${Turbo-E-Bike}
+    Click Element     ${Turbo E-Bike}
     Sleep   3s
```
Comment (Tatianafortech) [inline keywords/common/bike_details_ui_step_def.robot:25]: se poate scoate Sleep ->

**Comment by Tatianafortech** [comment]
De facut Handle pentru Add bike : (aici avem doar pt add your first bike)
- if bike present -> tap
- if bike not present but other present - add
- > maybe new ticket to add

**Comment by Tatianafortech** [comment]
bring up to date with main
cleanup la Settings -> resources
scos comentarii
uncoment TC: #Happy flow add remove bike
allow -> allow companion

**Comment by andreienea1990** [comment]
Sa repari te rog pylint:
test_runner.py:46:0: C0304: Final newline missing (missing-final-newline)

**Comment by andreienea1990** [comment]
Sa rezolvi conflictele te rog

**Comment by StefanaIvascu** [comment]
[laugh]         Stefana-Beatrice Ivascu reacted to your message:
________________________________

**Comment by andreienea1990** [comment]
This branch has conflicts that must be resolved
Use the [web editor](https://github.com/andreienea1990/mobileAutomation_appium/pull/18/conflicts) or the  to resolve conflicts.
Conflicting files
test_runner.py

**Comment by Tatianafortech** [comment]
please ignore files - do not change 

keywords/android/open_app.robot
keywords/iOS/open_app.robot
resources/common/
resources/android/settings.robot
resources/iOS/settings.robot

**Comment by Tatianafortech** [comment]
Alt branch a fost creat - pt demo - acest PR va fi inchis
https://github.com/andreienea1990/mobileAutomation_appium/pull/20

**Pull request closed without merging**